libs/domain/product/card/src/card.component.ts

```diff
@@ -95,7 +95,8 @@ export class ProductCardComponent extends ProductMixin(
   protected $link = computed(() =>
     this.semanticLinkService.get({
       type: RouteType.Product,
-      id: this.$product()?.sku,
+      id: featureVersion <= '1.4' ? this.$product()?.sku : undefined,
```

Comment:
```suggestion
      id: featureVersion < '1.4' ? this.$product()?.sku : undefined,
```

or stick to our current convention - not saying it's the right one ;)